analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -271,7 +271,7 @@ class Pip(
             // What pydep actually returns as "repo_url" is either setup.py's
             // - "url", denoting the "home page for the package", or
             // - "download_url", denoting the "location where the package may be downloaded".
-            // So the best we can do is to map this the project's homepage URL.
+            // So the best we can do is to map the project's homepage URL.
```

Comment:
Actually, I believe there was just a "to" missing after "this". Or maybe yet better "map it to".

Reply:
oh true.

Reply:
Done.